system/Filters/Filters.php

```diff
@@ -66,42 +65,81 @@ class Filters
     protected $initialized = false;
 
     /**
-     * The processed filters that will
-     * be used to check against.
+     * The filter list to execute for the current request (URI path).
      *
+     * This property is for display. Use $filtersClass to execute filters.
      * This does not include "Required Filters".
      *
-     * @var array<string, array>
+     * [
+     *     'before' => [
+     *         'alias',
+     *         'alias:arg1',
+     *         'alias:arg1,arg2',
+     *     ],
+     *     'after'  => [
+     *         'alias',
+     *         'alias:arg1',
+     *         'alias:arg1,arg2',
+     *     ],
+     * ]
+     *
+     * @var array{
+     *     before: list<string>,
+     *     after: list<string>
+     * }
      */
     protected $filters = [
         'before' => [],
         'after'  => [],
     ];
 
     /**
-     * The collection of filters' class names that will
-     * be used to execute in each position.
+     * The collection of filters' class names to execute for the current request
```

Comment:
filters' OR filters?

Reply:
I'm not sure it is grammatically correct or not,
but I have changed the sentence. https://github.com/codeigniter4/CodeIgniter4/pull/8977/commits/c0b1d8ac876a47e9cb5a06a70bdd59e87bf92e5d

Reply:
Thanks Kenjis.